tests/validation/config.json

```diff
@@ -406,7 +406,8 @@
             "title": "Self shadowing",
             "scriptToRun": "/Demos/SelfShadowing/shadows.js",
             "functionToCall": "CreateShadowsTestScene",
-            "referenceImage": "selfShadowing.png"
+            "referenceImage": "selfShadowing.png",
+            "excludedEngines": ["webgpu"]

```

Comment:
Was this one not ok before ?

Reply:
It was before the support for linear filtering is gone for float textures.

Now that test does not pass anymore because the ESM technic depends on the format of the texture (which is normally 32 bits float but is downscaled to 16 bits float in WebGPU).